nixos/modules/services/monitoring/ups.nix

```diff
@@ -382,7 +399,6 @@ let
         );
         NOTIFYCMD = lib.mkDefault "${pkgs.nut}/bin/upssched";
         POWERDOWNFLAG = lib.mkDefault "/run/killpower";
-        RUN_AS_USER = "root"; # TODO: replace 'root' by another username.
```

Comment:
In the previous hunk, RUN_AS_USER is set to nutmon. Why isn't this set here too? Feels like doc and impl. disagrees.

Reply:
Ah, good catch. I missed that one when switching to the `-u` option.